eslint/eslint-ts-rules-extra.mjs

```diff
@@ -390,4 +391,8 @@ export const extraRules = {
   // Unicorn
   "unicorn/filename-case": ["error", {case: "kebabCase"}],
   "unicorn/prefer-array-find": ["error"],
+  "unicorn/no-useless-spread": ["error"],
+
+  // SonarJS - additional rules
+  "sonarjs/no-duplicate-string": ["error", {"threshold": 5}],
```

Comment:
There is SonarJS above, lets group them together please

Reply:
I see we have this rule already in `eslint-ts-rules-overrides.mjs` as `"off"` - so I'll remove it from here.